lib/parsers/project-assets-json-parser.ts

```diff
@@ -0,0 +1,78 @@
+import * as _ from 'lodash';
+
+export interface PkgTree {
+  name: string;
+  version: string;
+  dependencies: {
+    [dep: string]: PkgTree;
+  };
+  targetFrameworks?: string[];
+}
+
+export interface ProjectAssetsJsonManifest {
+  targets: {
+    [target: string]: {
+      [name: string]: {
+        type: string,
+        dependencies?: {
+          [deps: string]: string;
+        };
+      };
+    },
+  };
+  project: {
+    restore: {
+      projectName: string,
+    },
+    version: string,
+  };
+}
+
+export function getProjectNameForProjectAssetsJson(manifestFile) {
```

Comment:
Why do we need to export these functions? Is it just because of the testing?

Reply:
Yup this is just for the tests

Reply:
In my opinion, than we don't need to test it at all. If it's a part of public interface just because of the testing, it usually means something strange with the design.

In this case, I believe it's enough to test that whole tree deep equals to the expectation. WDYT?

Reply:
We can remove it if we don't think that the testing the function is valuable, it was useful for me to write the test for the function with the stubs in place first and then extending to test for the entire expected tree.

Reply:
Removed the exporting `getProjectNameForProjectAssetsJson` and `getProjectVersionForProjectAssetsJson` and removed the unit tests